src/redux/store.jsx

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, please make sure that you include your slice's reducers in your store! Currently, your store doesn't know anything about Books slice or Categories slice! 🤓 🔧 

```suggestion
  reducer: { books: booksReducer, categories: categoriesReducer },
```